src/dragonball/src/device_manager/mod.rs

```diff
@@ -591,14 +591,25 @@ impl DeviceManager {
             .map_err(|_| StartMicroVmError::EventFd)?;
 
         info!(self.logger, "init console path: {:?}", com1_sock_path);
-        if let Some(path) = com1_sock_path {
-            if let Some(legacy_manager) = self.legacy_manager.as_ref() {
-                let com1 = legacy_manager.get_com1_serial();
-                self.con_manager
-                    .create_socket_console(com1, path)
-                    .map_err(StartMicroVmError::DeviceManager)?;
+        match com1_sock_path {
+            Some(path) => {
+                if path == "None" {
+                    info!(
+                        _ctx.logger(),
```

Comment:
why `_ctx.logger()` is used here?
BTW, `_` in Rust means this parameter is not used in the function, but you use `ctx` here.

Reply:
:smile:  Is ok to modify the `_ctx` to `ctx`?

Reply:
I think there is no need here to print the log.
Also, path == "None" looks strange in Rust, please help to see if this could be better